okta/FactorsClient.py

```diff
@@ -47,7 +47,7 @@ def get_available_questions(self, user_id):
         response = ApiClient.get_path(self, '/{0}/factors/questions'.format(user_id))
         return Utils.deserialize(response.text, Question)
 
-    def enroll_factor(self, user_id, factor_enroll_request, update_phone=None):
+    def enroll_factor(self, user_id, factor_enroll_request, update_phone=True, activate=False):
```

Comment:
Why did we change the default of `update_phone` to True here? Is that required for `activate` to work?

Reply:
no, but on the documentation this parameter should be a boolean, i changed for the consistence.